### PR TITLE
[14.0][IMP] fieldservice_recurring: Add equipment to recurrence

### PR DESCRIPTION
**Pull request by BojanOD**
Equipment `Many2many` field in the `fsm.recurring` model.

### CODE REVIEW

**Review by yankinmax — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

File: fieldservice_recurring/views/fsm_recurring.xml [hunk 1]
```diff
@@ -71,6 +71,7 @@
                                 groups="fieldservice.group_fsm_team"
                             />
                             <field name="person_id" />
+                            <field name="equipment_ids" widget="many2many_tags" />
```
Comment (max3903) [inline fieldservice_recurring/views/fsm_recurring.xml:74]:
```suggestion
                            <field name="equipment_ids" widget="many2many_tags" groups="fieldservice.group_fsm_equipment"/>
```

**Review by marcos-mendez — APPROVED** [review]
Thumbs Up

**Pull request merged**